src/javascript/audio.js

```diff
@@ -86,3 +127,25 @@ export default class Audio {
     this.player?.rate(rate);
   }
 }
+
+function shuffle(array) {
+  //ref https://stackoverflow.com/questions/2450954/
+
+  let currentIndex = array.length,
+    temporaryValue,
+    randomIndex;
+
+  // While there remain elements to shuffle...
+  while (0 !== currentIndex) {
+    // Pick a remaining element...
+    randomIndex = Math.floor(Math.random() * currentIndex);
+    currentIndex -= 1;
+
+    // And swap it with the current element.
+    temporaryValue = array[currentIndex];
+    array[currentIndex] = array[randomIndex];
+    array[randomIndex] = temporaryValue;
```

Comment:
```suggestion
    [array[currentIndex], array[randomIndex]] = [array[randomIndex], array[currentIndex]];
```

We rollin' with ES6+ now :D